v2/src/modules/tokens/ModuleTokensInfo.vue

```diff
@@ -1,167 +1,143 @@
+                            icon="arrow_upward"
+                            @click="sortTokens(KEY_VOLUME, 'desc')"
+                        />
+                        <app-btn-icon
+                            v-if="state.sortDirection === 'desc'"
+                            class="sort-button"
+                            :class="{ 'sort-button--active': state.sortKey === KEY_VOLUME }"
+                            icon="arrow_downward"
+                            @click="sortTokens(KEY_VOLUME, 'asc')"
+                        />
+                    </v-row>
+                </v-col>
+                <v-col md="2" class="d-none d-md-block">
+                    <v-row align="center" class="ma-0">
+                        <p>Market Cap</p>
+                        <app-btn-icon
+                            v-if="state.sortDirection === 'asc'"
+                            class="sort-button"
+                            :class="{ 'sort-button--active': state.sortKey === KEY_MARKET_CAP }"
+                            icon="arrow_upward"
+                            @click="sortTokens(KEY_MARKET_CAP, 'desc')"
+                        />
+                        <app-btn-icon
+                            v-if="state.sortDirection === 'desc'"
+                            class="sort-button"
+                            :class="{ 'sort-button--active': state.sortKey === KEY_MARKET_CAP }"
+                            icon="arrow_downward"
+                            @click="sortTokens(KEY_MARKET_CAP, 'asc')"
+                        />
+                    </v-row>
+                </v-col>
+                <v-col sm="2" md="2" lg="1"> Watchlist </v-col>
+            </v-row>
+            <v-divider class="my-0 mt-md-4 mx-n4 mx-sm-n6" />
+            <template v-if="!loadingCoinData">
+                <div v-if="tokensInPage.length > 0">
+                    <div v-for="token in tokensInPage" :key="token.symbol">
+                        <token-market-info-table-row :token="token" :favorite-tokens="favoriteTokens" @set-favorite="setFavoriteTokens" />
+                    </div>
+                    <app-intersect v-if="totalPages > 1" @intersect="loadMoreData">
+                        <div class="skeleton-box rounded-xl mt-1 my-4" style="height: 24px"></div>
+                        <v-divider />
+                    </app-intersect>
+                </div>
+                <div v-else>
+                    <v-row justify="center" class="my-0">
+                        <v-col md="10" class="bg-background rounded-lg mt-10 mb-8 py-12 text-center">
+                            <v-icon>info</v-icon>
+                            No results found. Please try again
                         </v-col>
                     </v-row>
-                    <v-divider />
                 </div>
-            </div>
-        </v-container>
-    </v-card>
+            </template>
+            <template v-else>
```

Comment:
there is a bug here. right now when you scroll to the end of the list the loading bar is still showing
<img width="1273" alt="Screen Shot 2022-09-12 at 1 34 37 PM" src="https://user-images.githubusercontent.com/16910687/189754614-f5885916-a2f3-4548-93dd-520e1e0c7111.png">